### ISSUE TITLE
Shortcut autocomplete ( ⌃Space ) doesnt work

### ISSUE THREAD

**Issue by MashaKorax** [dbeaver/dbeaver#19873]
### Description

 
![dbeaver_b12ERHM6VS](https://user-images.githubusercontent.com/84867187/236191730-c937462c-7b5b-4ec9-b142-cc58a6b2d3de.png)
 Im trying to use Shortcut autocomplete ( ⌃Space ) on macOs for filter field

Expected result: Pop-up with Columns Names from the current Table is shown
Actual result : nothing happens (on windows Ctrl+Space do the same and it works)

### DBeaver Version

23.0.4

### Operating System

macOs monterey 12

### Database and driver

Vertica

**Comment by Matvey16** [comment]
I don't think it is actually a bug. That happens because MacOS understands ^Space as language change.

Workaround for this is to disable this MacOS shortcut in preferences or change the content assist shorcut in DBeaver.
